fix(bcdb): search and replace methods

bearmit opened the pull request:
Cell values were not maching vocabulary, Dylan helped me. We added a new vocab "sans".

bearmit committed:
fix(bcdb): search and replace methods

Cell values were not maching vocabulary, Dylan helped me. We added a new
vocab "sans".

bearmit commented on scripts/typescript/src/bcdb/bcdb.ts:
```diff
@@ -208,26 +208,19 @@ export class BCDBLoader {
             unit: null,          
           });
 
-          // can be: "saxs", "tem", or "rheology"
-          let phase_method = row[Column.phase_method]?.toLowerCase();
-
-          // The custom vocabulary does not allow "rheology"
-          // Replacing with "rheometer".
-          if(phase_method === "rheology")  phase_method = "rheometer";
-
```
I removed this because the XLSX now contains valid and lower case values.

nh916 replied:
nice! cleaning up code as you got! love it!

nh916 commented:
> We added a new vocab "sans"

is this also added on the DB Schema and database?

We need to be sure every vocabulary word we add currently is replicated across all places needed so we don't encounter any errors

nh916 reviewed (approved):
I don't know this code base too well, but from what I can see it looks fine

bearmit commented:
> > We added a new vocab "sans"
> 
> is this also added on the DB Schema and database?
> 
> We need to be sure every vocabulary word we add currently is replicated across all places needed so we don't encounter any errors

A pull request is pending (accepted by Dylan, waiting for Brilant to merge and to deploy). In order to generate the final JSON I have to wait for it. But this PR does not contain the final file. So I'll merge this now.
